### PR TITLE
Update data_controller.proto

### PR DESCRIPTION
**Pull request by DJKirkman**
We have removed the title field from the citation table so can just send citation bodies as repeated strings. We have also added abstarcts to the content field so need a new parameter for this.

### CODE REVIEW

**Comment by DJKirkman** [comment]
@KyleCotton can you double check this plz and test it complies.

File: proto/data_controller.proto [hunk 1]
```diff
@@ -15,6 +15,12 @@ message ArticleRequest {
     string articleTitle = 2;
 }
 
+message DumpRequest {
+    string domain = 1;
+    string articleTitle = 2;
```
Comment (KyleCotton) [inline proto/data_controller.proto:20]: When we are updating a `dumpId` why do we need the `domain` and `articleTitle`?

File: proto/data_controller.proto [hunk 2]
```diff
@@ -5,7 +5,7 @@ package data_controller;
 service DataController {
     rpc getLastUpdatedDateTime(ArticleRequest) returns (LastUpdatedDateResponse) {}
     rpc createDump(CreateDumpRequest) returns (DumpId) {}
-    rpc updateDumpId(ArticleRequest) returns (UpdateDumpIdResponse) {}
+    rpc updateDumpId(DumpRequest) returns (UpdateDumpIdResponse) {}
```
Comment (KyleCotton) [inline proto/data_controller.proto:8]: I think we should update the name of this method to `completeDump` since that is the only time it is used.

Comment (KyleCotton) [reply proto/data_controller.proto:8]: Actually looking at this.
What is the `updateDumpId` for?

File: proto/data_controller.proto (hunk 1 above)
Comment (DJKirkman) [reply proto/data_controller.proto:20]: So we can identify an article from the data we have available from the dump file (we don't have the articleId from the dumpfile)

File: proto/data_controller.proto (hunk 2 above)
Comment (DJKirkman) [reply proto/data_controller.proto:8]: Does this make sense now?

**Review by KyleCotton — APPROVED** [review]
Yes, everything is working, I have generated the new files and added them to this PR.

**Pull request merged**